File: 30 day code c++/day2.cpp
#include <bits/stdc++.h>

using namespace std;

// Complete the solve function below.
void solve(double meal_cost, int tip_percent, int tax_percent) {
 float tip,tax;
 tax=(float)meal_cost*tax_percent/100;
 tip=(float)meal_cost*tip_percent/100;
 int totalCost;
 totalCost=round(meal_cost + tip + tax);
 printf("%d",totalCost);
}

int main()
{
    double meal_cost;
    cin >> meal_cost;
    cin.ignore(numeric_limits<streamsize>::max(), '\n');

    int tip_percent;
    cin >> tip_percent;
    cin.ignore(numeric_limits<streamsize>::max(), '\n');

    int tax_percent;
    cin >> tax_percent;
    cin.ignore(numeric_limits<streamsize>::max(), '\n');

    solve(meal_cost, tip_percent, tax_percent);

    return 0;
}

File: 30 days of code JAVA/day2.java
import java.util.*;
public class day2 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        double meal_cost=sc.nextDouble();
        int tip_percent=sc.nextInt();
        int tax_percent=sc.nextInt();
        solve(meal_cost,tip_percent,tax_percent);
    }
    public static void solve(double meal_cost,int tip_percent,int tax_percent){
        float tip,tax;
        int totalCost;
        tax=(float)meal_cost*tax_percent/100;
        tip=(float)meal_cost*tip_percent/100;
        totalCost=(int)(meal_cost+tip+tax);
        System.out.println(totalCost);
    }
}
